### PR TITLE
BOLT-1: Minor line edits

### PR DESCRIPTION
**Pull request by shannona**
These are minor edits for BOLT-1, constrained to punctuation and minor word changes. In a few places clauses were rearranged. The changes were grammatical or for clarity.

### CODE REVIEW

**Comment by rustyrussell** [comment]
So, we agreed last meeting to get two pairs of eyes before applying changes like this.  So I'd hoped to get @toadlyBroodle to review your changes and vice versa, but I can't assign them as reviewers.  I've tried inviting you both to be read-only collaborators, in this hope that will let me use the Github UI to ping you each for reviews.

Meanwhile, I'm reviewing; mainly to check contents, as I'm underqualified to comment on broader style issues :)

File: 01-messaging.md [hunk 1]
```diff
@@ -22,49 +22,49 @@ All data fields are big-endian unless otherwise specified.
...
+The size of the message is required by the transport layer to fit into a 2-byte unsigned int; therefore, the maximum possible size is 65535 bytes.
 
 A node:
-  - MUST ignore any additional data within a message, beyond the length it expects for that type.
+  - MUST ignore any additional data within a message beyond the length that it expects for that type.
   - upon receiving a known message with insufficient length for the contents:
     - MUST fail the channels.
-  - that understands an option (each is numbered):
+  - that understands a numbered option:
     - MUST include all the fields annotated with that option.
```
Comment (rustyrussell) [inline 01-messaging.md]: Hmm, let's just remove "(each is numbered)", since they aren't any more.  I'll typo fix this separately.

**Comment by shannona** [comment]
I've now added in BOLT-0, where there was some more extensive rewriting of some sentences for clarity. Particularly check the glossary where I (1) wrote definitions for nodes and peers that had been missing; (2) changed Commitment revocation key to Commitment revocation secret key to match the rest of the document; and (3) broke apart payment hash and payment preimage definitions.

File: 00-introduction.md [hunk 2]
```diff
@@ -25,95 +25,109 @@ This is version 0.
...
+   * The *HTLC* contains the payment hash, which is the hash of the
+    *payment preimage*. 
+
+
+* *Payment preimage*: 
+   * Proof that payment has been received, held by
+    the final recipient, who is the only person who knows this
+    secret. The final recipient releases the preimage in order to
+    release funds. The payment preimage is hashes as the *payment hash*
+    in the *HTLC*.
```
Comment (rustyrussell) [inline 00-introduction.md]: s/is hashes/is hashed/

**Comment by rustyrussell** [comment]
You'll also need to append the word "Bitcoins" to the spellcheck wordlist in ".aspell.en.pws".

**Comment by shannona** [comment]
Typo fix should be in there now.

**Pull request merged**